bot/exts/utils/snekbox/_eval.py

```diff
@@ -50,8 +50,16 @@ def to_dict(self) -> dict[str, list[str | dict[str, str]]]:
         return {
             "args": self.args,
             "files": [file.to_dict() for file in self.files],
+            "executable_path": self.executable_path(),
         }
 
+    def executable_path(self) -> str:
+        if self.version == "3.12":
+            return "/snekbin/python/3.12/bin/python"
+        if self.version == "3.13":
+            return "/snekbin/python/3.13/bin/python"
+        return "/snekbin/python/default/bin/python"
```

Comment:
Nit: this could use string formatting instead of if/else.

Reply:
Yeah, not sure why I made it so complicated. https://github.com/python-discord/bot/compare/81efc67838f86cb8fb74f66a75b3e84f27d194d6..60e29b1a15a2f026a71114fed7bb2f2f93b2548d

This also showed a test I overlooked, which I also corrected